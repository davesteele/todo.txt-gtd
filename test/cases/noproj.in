# _None

new taskproj @computer
