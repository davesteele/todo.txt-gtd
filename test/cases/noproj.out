# X-None

new taskproj @computer
